glamour.go

```diff
@@ -122,7 +133,7 @@ func WithStandardStyle(style string) TermRendererOption {
 // WithAutoStyle sets a TermRenderer's styles with either the standard dark
 // or light style, depending on the terminal's background color at run-time.
 func WithAutoStyle() TermRendererOption {
-	return WithStandardStyle("auto")
+	return WithStandardStyle(string(AutoStyle))
```

Comment:
I believe this `string` conversion is superfluous

```suggestion
	return WithStandardStyle(AutoStyle)
```

Reply:
Ah good catch; that's WIP residue. Will remove.